web/src/lib/components/photos-page/actions/download-action.svelte

```diff
@@ -5,11 +5,23 @@
   import { getAssetControlContext } from '../asset-select-control-bar.svelte';
   import { mdiCloudDownloadOutline } from '@mdi/js';
 
-  export let filename = 'immich.zip';
   export let menuItem = false;
 
   const { getAssets, clearSelect } = getAssetControlContext();
 
+  const formatDate = () => {
```

Comment:
Can we use Luxon to format the date string instead of manually doing it?

https://github.com/moment/luxon/blob/master/docs/formatting.md#toformat

Reply:
Thanks @alextran1502. 
I have updated the code.